bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/entity/PhysicalPostalAddress.kt

```diff
@@ -113,4 +113,30 @@ class PhysicalPostalAddress(
      */
     @Column(name = "phy_door")
     var door: String?
-)
+) {
+
+    override fun equals(other: Any?): Boolean {
```

Comment:
You can make this a data class and get rid of `equals`.
If not, take care that you also override `hashCode` together with `equals` in a compatible way!